Dockerfile

```diff
@@ -61,7 +61,7 @@ RUN --mount=type=bind,from=generate,source=/out,target=_site <<EOF
     --disable-external \
     --internal-domains="docs.docker.com,docs-stage.docker.com,localhost:4000" \
     --file-ignore="/^./_site/engine/api/.*$/,./_site/registry/configuration/index.html" \
-    --url-ignore="/^/docker-hub/api/latest/.*$/,/^/engine/api/v.+/#.*$/,/^/glossary/.*$/" > /results 2>&1
+    --url-ignore="/^/docker-hub/api/latest/.*$/,/^/engine/api/v.+/#.*$/,/^/glossary/.*$/,/^/develop/develop-images/build_enhancements/#new-docker-build-secret-information$/" > /results 2>&1
```

Comment:
Temporarily ignore build_enhancements linking to internal hash new-docker-build-secret-information in htmlproofer while waiting for changes in Dockerfile reference on BuildKit repo:

```
#0 0.096 - ./_site/engine/reference/builder/index.html
#0 0.096   *  linking to internal hash #new-docker-build-secret-information that does not exist (line 2547)
#0 0.096      <a href="/develop/develop-images/build_enhancements/#new-docker-build-secret-information">[“build images with BuildKit”](https://github.com/develop/develop-images/build_enhancements/#new-docker-build-secret-information)</a>
```

Reply:
Opened https://github.com/moby/buildkit/pull/3220